.travis.yml

```diff
@@ -39,6 +39,10 @@ script:
       mvn clean install -B -U
     fi
 
+# enable codecov-io
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Actually, it's not that straightforward. Pull requests from committers have access to secrets. I need to investigate this a bit more to find a workable solution.